Misc/NEWS.d/next/Library/2020-09-20-15-14-05.bpo-41810.7l8lyV.rst

```diff
@@ -0,0 +1,2 @@
+:data:`types.EllipsisType`, :data:`types.EllipsisType` and
+:data:`types.NoneType` have been reintroduced.
```

Comment:
We should mention the reason behind this reintroduction: for type checkers which can't use `type(Ellipsis)`, etc.

Reply:
I added a few words about the motivation in https://github.com/python/cpython/pull/22336/commits/370149882d2cea95f2fed7c732bcbb6ae343a3fe.